admin/spec/service/check-window.service.spec.js

```diff
@@ -139,4 +140,34 @@ describe('check-window.service', () => {
       })
     })
   })
+
+  describe('#getCurrentCheckWindowsAndCountForms', () => {
+    let fetchCurrentCheckWindowsStub
+
+    beforeEach(() => {
+      fetchCurrentCheckWindowsStub = sandbox.stub(checkWindowDataService, 'fetchCurrentCheckWindows').resolves(checkWindowsMock)
+      service = setupService(function () { return Promise.resolve(checkWindowsAndCountFormsMock) })
```

Comment:
Shouldn't this service be the service you are testing and not a mock?

Reply:
Yes, that's correct. Fix done.